From a092650bfece8a67cddffc7983dce06146f81ffb Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 6 Mar 2022 11:03:11 +0000
Subject: [PATCH] libstdc++: fix incomplete type of blink::WebURLLoaderFactory

std::unique_ptr of libstdc++ uses sizeof() which requires full
definition of blink::WebURLLoaderFactory in blink::EmptyChromeClient.
---
 third_party/blink/renderer/core/loader/empty_clients.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/blink/renderer/core/loader/empty_clients.h b/third_party/blink/renderer/core/loader/empty_clients.h
index d91e67a..e23de75 100644
--- a/third_party/blink/renderer/core/loader/empty_clients.h
+++ b/third_party/blink/renderer/core/loader/empty_clients.h
@@ -43,6 +43,7 @@
 #include "third_party/blink/public/mojom/input/focus_type.mojom-blink-forward.h"
 #include "third_party/blink/public/platform/web_spell_check_panel_host_client.h"
 #include "third_party/blink/public/platform/web_url_loader.h"
+#include "third_party/blink/public/platform/web_url_loader_factory.h"
 #include "third_party/blink/renderer/core/core_export.h"
 #include "third_party/blink/renderer/core/frame/local_frame_client.h"
 #include "third_party/blink/renderer/core/frame/remote_frame_client.h"
-- 
2.34.1

